File: practices/old_enough.py
#Ah 7th old enough

age = int(input("what is your age?\n"))
if age >= 18:
    print("you can vote!")
elif age >= 16:
    print("you can drive!")
elif age >= 15:
    print("you can get a learners permit!")
elif age >= 4:
    print("you can go to school!")
else:
    print("you are not old enough!")

File: c_programming/practces/old_enough.c
//Ah 7th old enough
#include <stdio.h>

int main(void){ 
    int age;
    printf("give me your age: \n");
    scanf("%d", &age);
    if (age >= 18){
        printf("you can vote!\n");
    }else if (age >= 16){
        printf("you can drive!\n");
    }else if (age >= 15){
        printf("you can get a learners permit!\n");
    }else if (age >= 4){
        printf("you can go to school!\n");
    }else{
        printf("you are not old enough!\n");
    }
    return 0;
}